config-dev.json

```diff
@@ -17,6 +17,11 @@
         "https://api.blockchain.info/eth/nodes/rpc"
       ],
       "shortName": "Ethereum",
+      "supportedBy": [
+        "DEFI",
+        "NATIVE",
+        "TRADING"
```

Comment:
I don't think we need `TRADING` here. It's defined per-product in `coin-definitions` / `assets` service